views/includes/line-graph.pug

```diff
@@ -13,7 +13,7 @@ else
 script.
 	Chart.defaults.global.elements.point.radius = 1;
 	var ctx = document.getElementById("#{graphData.id}").getContext('2d');
-	const gradient = ctx.createLinearGradient(0, 0, 0, 400);
+	var gradient = ctx.createLinearGradient(0, 0, 0, 400);
```

Comment:
why var?

Reply:
because it was before, I've changed it to `const` but it's imported multiple time and theres problem with already defined variable.